cli/cli_test.go

```diff
@@ -365,8 +370,25 @@ func testInit(t *testing.T) {
 	nomadAddr = os.Getenv("NOMAD_ADDR")
 	_ = os.Setenv("NOMAD_ADDR", "http://127.0.0.1:4646")
 
+	_, err := os.Stat(path.Join(cache.DefaultCachePath(), cache.DefaultRegistryName, "simple_service@latest"))
```

Comment:
Just out of curiosity, why do we need this new block?

Reply:
When I ran the tests with an empty packs folder, it would not download the latest registry without adding this case.  This caused all of the tests that used `simple_service@latest` to fail. I could have worked around it by cloning default first myself, but that seems to be a surprising prerequisite to running the tests.

Reply:
😮  

That seems like something to file an issue for, yeah?